hikari/impl/special_endpoints.py

```diff
@@ -898,25 +935,28 @@ def build(self, entity_factory: entity_factory_.EntityFactory, /) -> data_bindin
 class CommandBuilder(special_endpoints.CommandBuilder):
```

Comment:
Should probably consider having two separate builder classes, one for Slash commands and one for menu commands. would give passive assertions that slash command specific fields aren't specified for slash commands or vice versa